watchOS/ContentView.swift

```diff
@@ -10,12 +10,26 @@ struct ContentView: View {
     
     @State private var showLoginSheet = false
     
+    private func syncCreditials() async throws {
+        let state = try await requestCredentialTransfer()
+        switch state {
+        case .isLogged(let username, let password, let studentType):
+            campusModel.forceLogin(username: username, password: password)
+            campusModel.studentType = studentType
+        case .notLogged:
+            campusModel.logout()
+        }
+    }
+    
     var body: some View {
         NavigationStack {
             if campusModel.loggedIn {
                 CampusContent()
+                    .task {
+                        try? await syncCreditials()
```

Comment:
Do we need to watch to request synchronization every time it is opened? Would it be better to just start a `transferUserInfo` request (which is always delivered) when the login state on iPhone changes?